engine/xml-config/src/main/java/org/datacleaner/job/JaxbJobReader.java

```diff
@@ -915,9 +919,50 @@ private void applyProperties(final ComponentBuilder builder,
                     final Converter<?> customConverter = configuredProperty.createCustomConverter();
                     final Object value = stringConverter.deserialize(stringValue, configuredProperty.getType(),
                             customConverter);
+               
+                    if (name.equals(CoalesceMultipleFieldsTransformer.PROPERTY_UNITS)) {
```

Comment:
If `FuseStreamsComponent` is using the same name it doesn't matter, but we may want to ensure that it is the correct type. We don't control the naming of all transformers.